service/storage_service.cc

```diff
@@ -2288,11 +2295,13 @@ void storage_service::run_bootstrap_ops(std::unordered_set<token>& bootstrap_tok
             }
             slogger.warn("bootstrap[{}]: Found pending node ops = {}, sleep 5 seconds and check again", uuid, pending_ops);
             sleep_abortable(std::chrono::seconds(5), _abort_source).get();
+
+            ctl.refresh_sync_nodes([this_node = get_broadcast_address()] (gms::inet_address node) {
```

Comment:
We can drop the filter here too.

But refresh_sync_nodes will revert what we do at line 2274 which adds this_node to sync_nodes. 

We need to call here:

```
ctl.refresh_sync_nodes();
ctl.sync_nodes.insert(get_broadcast_address());
```

Reply:
But it will be re-added by line 2274 when we loop back.